### PR TITLE
mailmap, organizationmap: add Dongsheng, Liuzhong, Pengcheng, Yang Affiliation

### PR DESCRIPTION
**Pull request by yangdongsheng**
Signed-off-by: Dongsheng Yang <dongsheng.yang@easystack.cn>

### CODE REVIEW

File: .organizationmap [hunk 1]
```diff
@@ -135,7 +135,10 @@ EISOO <contact@eisoo.com> lu.shasha <lu.shasha@eisoo.com>
 Enproduktion GmbH <contact@produktion.io> Malte Fiala <maltefiala@produktion.io>
 EPAM <contact@epam.com> Andrey Kuznetsov <Andrey_Kuznetsov@epam.com>
 Exalead <contact@exalead.com> <roche+git@exalead.com>
+Easystack <contact@easystack.cn> chenliuzhong <liuzhong.chen@easystack.cn>
```
Comment (joscollin) [inline .organizationmap]: It looks like this is a github username `chenliuzhong`. Please provide your actual name here, which is usually a display name in the Email. It should be in the alphabetical order with in the group `Easystack` too.

File: .organizationmap [hunk 2]
```diff
@@ -135,7 +135,10 @@ EISOO <contact@eisoo.com> lu.shasha <lu.shasha@eisoo.com>
...
+Easystack <contact@easystack.cn> kungf <yang.wang@easystack.cn>
```
Comment (joscollin) [inline .organizationmap]: ditto

File: .organizationmap [hunk 3]
```diff
@@ -135,7 +135,10 @@ EISOO <contact@eisoo.com> lu.shasha <lu.shasha@eisoo.com>
...
 Easystack <contact@easystack.cn> Dongsheng Yang <dongsheng.yang@easystack.cn>
+Easystack <contact@easystack.cn> PCzhangPC <pengcheng.zhang@easystack.cn>
```
Comment (joscollin) [inline .organizationmap]: ditto

File: .mailmap [hunk 4]
```diff
@@ -74,6 +74,7 @@ Clement Lebrun <clement.lebrun.31@gmail.com> <clem_noob@hotmail.fr>
 Colin P. McCabe <colinm@hq.newdream.net> <cmccabe@alumni.cmu.edu>
 Colin P. McCabe <colinm@hq.newdream.net> <cmccabe@fatty.ops.newdream.net>
 Colin Walters <walters@redhat.com> <walters@verbum.org>
+chenliuzhong <liuzhong.chen@easystack.cn>
```
Comment (joscollin) [inline .mailmap]: It looks like this is a github username `chenliuzhong`. Please provide your actual name here in the alphabetical order. 

@tchaikov Could you please suggest if this entry is necessary as `chenliuzhong` is having only one Email ID ? Thanks.

File: .mailmap [hunk 5]
```diff
@@ -225,6 +226,7 @@ Ken Dreyer <kdreyer@redhat.com>
 Kévin Caradant <kevin.caradant@gmail.com>
 Kiseleva Alyona <akiselyova@mirantis.com>
 Kongming Wu <wu.kongming@h3c.com>
+kungf <yang.wang@easystack.cn>
```
Comment (joscollin) [inline .mailmap]: ditto

File: .mailmap [hunk 6]
```diff
@@ -287,6 +289,7 @@ Noah Watkins <nwatkins@redhat.com> <jayhawk@cs.ucsc.edu>
 Noah Watkins <nwatkins@redhat.com> <noahwatkins@gmail.com>
 Orit Wasserman <owasserm@redhat.com> <owassern@redhat.com>
 Orit Wasserman <owasserm@redhat.com> <owassrm@redhat.com>
+PCzhangPC <pengcheng.zhang@easystack.cn>
```
Comment (joscollin) [inline .mailmap]: ditto

File: .mailmap (hunk 4 above)
Comment (theanalyst) [reply .mailmap]: Yeah keeping an entry in mailmap always helps when there are future mistakes etc.

Comment (joscollin) [reply .mailmap]: @theanalyst oh okay, I have seen many people with single emails in .mailmap.

**Comment by yangdongsheng** [comment]
@joscollin thanx for your review. Please take a look this version. The first two commits are sorting the mailmap and organizationmap by ```sort``` command.

**Comment by yangdongsheng** [comment]
https://github.com/ceph/ceph/pull/17567 @joscollin

**Comment by yangdongsheng** [comment]
@theanalyst @joscollin  drop the sorting commits now. Lets sort them later as @theanalyst mentioned in #17567

**Pull request merged**